supertokens_python/__init__.py

```diff
@@ -30,9 +30,36 @@ def init(app_info: InputAppInfo,
          recipe_list: List[Callable[[supertokens.AppInfo], RecipeModule]],
          mode: Union[Literal['asgi', 'wsgi'], None] = None,
          telemetry: Union[bool, None] = None):
+    """init.
+
+    Parameters
+    ----------
+    app_info : InputAppInfo
+        app_info
+    framework : Literal['fastapi', 'flask', 'django']
+        framework
+    supertokens_config : SupertokensConfig
+        supertokens_config
+    recipe_list : List[Callable[[supertokens.AppInfo], RecipeModule]]
+        recipe_list
+    mode : Union[Literal['asgi', 'wsgi'], None]
+        mode
+    telemetry : Union[bool, None]
+        telemetry
```

Comment:
I really see no point in writting this when the generated docs has these very clearly mentioned as part of the function signature anyway. 

COuld you please explain the benefit of this?

Reply:
So in the example of flask that you gave, the function signature is good and there is a lot of description added about each param (and not just the types). In our current API docs (see [this](https://supertokens.com/docs/python/recipe/emailpassword/asyncio/index.html)), you can see that the input and its types are very clearly shown already. So in the docs string, I would expect a good description of the params, and what they do etc.. not just the types (cause you can see the types already).

Am I totally missing the point here?

Reply:
> So in the docs string, I would expect a good description of the params, and what they do etc.. not just the types (cause you can see the types already).

That is correct.

Also, it won't be possible to add the detailed description immediately to all of the classes and functions' docstrings, this PR just adds the template documentation for each class/method/function which needs to be fixed by someone in the future.

Also to test the document coverage, tools like [docstr-coverage](https://github.com/HunterMcGushion/docstr_coverage) and [interrogate](https://interrogate.readthedocs.io/en/latest/) can be used.

In addition to that, Vscode extensions like [autoDocstring](https://marketplace.visualstudio.com/items?itemName=njpwerner.autodocstring) can be used to generate docstrings during the development.

Reply:
ok thanks! Will have a look.